Connectors/Rhino/Speckle.Connectors.GrasshopperShared/Operations/Send/GrasshopperSendOperation.cs

```diff
@@ -79,10 +90,10 @@ GrasshopperBlockPacker blockPacker
           Unwrap(collWrapper, colorPacker, materialPacker, blockPacker);
           break;
 
-        // NOTE: order is super important since SpeckleBlockInstanceWrapper inherits from SpeckleObjectWrapper
-        // this is fragile and not nice.
+        // NOTE: order is important; since SpeckleBlockInstanceWrapper inherits from SpeckleObjectWrapper, this needs to come first
```

Comment:
Lines 97 and 98 seem to apply to generic speckleobjectwrapper.
It's probably more robust to combine instance wrapper and object wrapper cases into a single object wrapper case, and add lines 101-116 as a separate method to process instances specifically inside that case (if objectwrapper is instancewrapper, process def).